Hackaton by Cesi (#82)
lunatech-labs/lunatech-blog: posts/2023-02-15-hackaton-made-by-cesi.adoc

> = Hackaton Made by Cesi

Author: The name of the event is "Hackathon Made by CESI" but maybe it's not very explicit in the article.... 

(I added the missing h thanks!)

> = Hackaton Made by Cesi

Reviewer: You're right about removing the word "made" but we cannot add the word "Apocalypse" as it would change the name of the event.

> = Hackaton Made by Cesi

Reviewer 2: OK. I had seen the hackathon flyer and it was saying "Hackathon Apocalypse" so I thought that this was the title. No problem then.

sachgar
v1.0, 2023-02-15
:title: hackaton-made-by-cesi
:lang: en
:tags: [hackaton, event, en]

> On January 28, 2023, the CESI engineering school in La Rochelle hosted a hackathon with an apocalyptic theme. 25 participants gathered in teams of five to imagine innovative solutions to deal with an emergency situation. As with any events, it was essential for us to find sponsors to ensure that the event will be successful. We naturally thought Lunatech would be a good fit for us. They kindly agreed to finance the staff's hoodies and to send us some goodies for all of the participants.

Reviewer: We can add a link to our school website. Where's the best place to put it?

> On January 28, 2023, the CESI engineering school in La Rochelle hosted a hackathon with an apocalyptic theme. 25 participants gathered in teams of five to imagine innovative solutions to deal with an emergency situation. As with any events, it was essential for us to find sponsors to ensure that the event will be successful. We naturally thought Lunatech would be a good fit for us. They kindly agreed to finance the staff's hoodies and to send us some goodies for all of the participants.

Author: Changes made

> On January 28, 2023, the CESI engineering school in La Rochelle hosted a hackathon with an apocalyptic theme. 25 participants gathered in teams of five to imagine innovative solutions to deal with an emergency situation. As with any events, it was essential for us to find sponsors to ensure that the event will be successful. We naturally thought Lunatech would be a good fit for us. They kindly agreed to finance the staff's hoodies and to send us some goodies for all of the participants.

Reviewer 2: > We can add a link to our school website. Where's the best place to put it?

Something like this:
```the [CESI engineering school](https://www.cesi.fr/) in La Rochelle```?

image::../media/2023-02-15-hackaton-made-by-cesi/staff.jpeg[Staff,640,480]

> The hackathon started with a presentation of the apocalyptic scenario, followed by a brainstorming session to generate ideas. In order to help participants in their search for ideas, different techniques were used such as the World Cafe. 

Reviewer: The apocalyptic scenario in details:
```
"Attention all participants of our hackathon! It is the 10th of January 2049! We have received alarming information that an apocalyptic catastrophe is about to happen in our city in the year 2050, according to the predictions of the doomsday clock and several scientists. Scientists estimate that it will happen in the next few months.

Fortunately, we have prepared a secure bunker to protect the participants of the event. However, it is our duty to prepare our city for this catastrophe. This is where you come in!

We need your help to find ways to protect the people and property of our city. You have 24 hours on 28 January to come up with an innovative and workable solution to this disaster.

We are counting on your ingenuity and determination to save our city. Good luck to you all!"
```

> The hackathon started with a presentation of the apocalyptic scenario, followed by a brainstorming session to generate ideas. In order to help participants in their search for ideas, different techniques were used such as the World Cafe. 

Reviewer: The wikipedia article is a great addition.

> This technique is great for brainstorming with an important number of groups, it goes as follows: each team has Post-it (a lot of them), a surface to stick them on and pens. During 5 minutes, each team put forth as many ideas as possible. The rule is quantity over quality. After 5 minutes, each team will go to the next team board, one member will stay to explain the idea to the other team. We can now repeat the first step for 5 minutes. The World Café should allow each team to visit at least two different boards.

Reviewer: I wanted to say that this technique is most useful with a large number of groups. 
I intended to try to keep my explanation general by using "a surface to stick them on" but I you think using "whiteboard" is better for the context it's fine by me.

> This technique is great for brainstorming with an important number of groups, it goes as follows: each team has Post-it (a lot of them), a surface to stick them on and pens. During 5 minutes, each team put forth as many ideas as possible. The rule is quantity over quality. After 5 minutes, each team will go to the next team board, one member will stay to explain the idea to the other team. We can now repeat the first step for 5 minutes. The World Café should allow each team to visit at least two different boards.

Author: _I think "The World Café should allow each" should be rephrased to "The World Café allows each"_ Done

> Participants had the chance to attend three exciting conferences on the topics of Low Tech, Women in Engineering and the Engineer of the Future during the morning. The objective was to encourage participants to integrate or build on these topics in the research and development of their ideas. The conferences brought a new perspective to the participants on the current and future challenges of engineering. Low Tech was presented as a creative and sustainable alternative to high-tech technologies for dealing with emergency situations. The place of women in engineering was discussed, highlighting the importance of diversity in engineering in order not to deprive ourselves of any skills and to build an inclusive as well as equitable solution. The engineer of the future was presented as being more ethical, sustainable and aware of the consequences of their actions.

Reviewer: They really were named conferences during the event. Two of them were more interactive (Kahoot and Guessing from an image the associated Low-Tech invention).

Lunch was a key moment of the hackathon, as participants were able to discuss and exchange ideas with their teammates. This strengthened their cohesion and collaboration, which contributed to the success of their projects.

image::../media/2023-02-15-hackaton-made-by-cesi/caterer.jpeg[Caterer,640,480]

The teams then went back to work to develop their ideas and transform them into a real innovative concept. The afternoon was also dedicated to the preparation of the final presentation during which the different teams had to face a jury in charge of establishing the ranking and an audience composed of all the coaches and parents.

image::../media/2023-02-15-hackaton-made-by-cesi/public.jpeg[Public,640,480]

> The first place team conceptualized a survival school to teach people the skills needed to cope with an apocalyptic situation. The second place team developed the idea of a virtual reality simulation to help people prepare mentally and physically for the apocalypse. And finally the team completing the podium made a self-sufficient dome system, which can provide food, water and energy for people in an emergency situation.

Reviewer 2: Maybe add something about the jury and how they came to this decision.
For example "After a few minutes/half hour the jury announced the results of the hackathon" to make the transition between the teams facing the jury and the results, more smooth.

> The day ended with a prize-giving ceremony for each team. The first team received airpods, the second team received wireless speakers, the third team received gift cards and the fourth and fifth team received chocolate sets made by a local company.

Reviewer 2: ```suggestion
The day ended with a prize-giving ceremony. The first team was awarded airpods, the second team received wireless speakers, the third team was given gift cards while the fourth and fifth team received chocolate sets made by a local company.
```

This is not a strict recommendation. Just giving ideas on verbs to use instead of repeating "received". :)
